devices/collect/hrtim/v1.yaml

```diff
@@ -0,0 +1,25 @@
+"HRTIM_TIM[BCDEF]":
```

Comment:
If this refers to the HRTIM found in the F334 then I believe that does not have timer F

Reply:
It is just rule for search. It applies rules only for founded peripherals. It warnings only if nothing has founded.

Reply:
You can see it in memory map comparison:
https://github.com/stm32-rs/stm32-rs-mmaps/compare/pr-1024-1734720a3f753ade57e0324215b1ba9a9cec7c38#diff-961b24096e627c8e5fbf2c34d8658c99a086687d00ecf7280b92055c1e37d96c

Reply:
Okay that makes sense, thanks